app/src/utils/AuthUtils.js

```diff
@@ -33,6 +34,11 @@ export const isEmailVerified = (userId) => {
     .catch((err) => Logger.log(err));
 };
 
+export const isDisposableEmail = (email) => {
+  const domain = email.split("@").reverse()[0];
```

Comment:
I am just trying to get the last element of the generated array. 
because javascript does not support negative indexing of arrays and doing this with `.length-1` was looking a little messy to me.